# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/config/gfxVars.h
# Commit: 4f5567292c37
# Full Hash: 4f5567292c37eedabac14b615f41c961cfc244f3
# Author: stransky <stransky@redhat.com>
# Date: 2022-02-24 03:51:41
# Regressor Bug: 1751987
# File Overlap Count: 1
# Description:
#   Bug 1751987 [Linux] Add VA-API feature r=aosmond
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D139301
# ==============================================================================

diff -r b7ea622c1205 -r 4f5567292c37 gfx/config/gfxVars.h
--- a/gfx/config/gfxVars.h	Wed Feb 23 11:47:06 2022 +0000
+++ b/gfx/config/gfxVars.h	Wed Feb 23 11:47:07 2022 +0000
@@ -84,6 +84,7 @@
   _(UseEGL, bool, false)                                           \
   _(DrmRenderDevice, nsCString, nsCString())                       \
   _(UseDMABuf, bool, false)                                        \
+  _(UseVAAPI, bool, false)                                         \
   _(WebRenderRequiresHardwareDriver, bool, false)                  \
   _(SupportsThreadsafeGL, bool, false)                             \
   _(OffscreenCanvasDomainAllowlist, nsCString, nsCString())